app/components/header.tsx

```diff
@@ -1,11 +1,78 @@
-import { Text, Center } from '@chakra-ui/react';
+import {
+  Text,
+  Heading,
+  Avatar,
+  Flex,
+  HStack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  IconButton,
+} from '@chakra-ui/react';
+import { TriangleUpIcon, LockIcon } from '@chakra-ui/icons';
+import { Link, useFetcher } from '@remix-run/react';
+
+import { useUser } from '~/utils';
 
 export default function Header() {
+  const user = useUser();
+  const fetcher = useFetcher();
   return (
     <header>
-      <Center>
-        <Text fontSize="2xl">Hi! I am the header</Text>
-      </Center>
+      <Flex
+        backgroundColor="brand.500"
+        height="70px"
```

Comment:
Is there any good reason to set static height for the header? Maybe just leave it `auto` and just add horizontal padding if you wanna increase the height?

Reply:
Can't we just set it to break points if we need to manipulate the value of the height? Something like this?
![image](https://user-images.githubusercontent.com/32577022/218173918-31d7954b-c074-4e5f-9cf7-2b98650cbcbd.png)

Reply:
I don't see any good reason why we would need to manipulate the value of the height, in a way that we set it statically, it could just stretch depending on its content height + you could put some vertical padding `px="2"` or something

Reply:
It is better to avoid settings up static height to avoid cases when your content's height is bigger than the container height, let's say if on a mobile device logo wouldn't fit in one line and would go for another one, but your static height is designed to fit only one line, so it would be overflown.

Reply:
And regarding setting padding (I actually made mistake, it should be something like `py="2"`, since we talk about the height) you could just set vertical padding for container and it would be alright for all screens, while with the height you need to adjust the height for all or some breakpoints

Reply:
Ohh that make sense, I'll remember that